packages/serverless-contracts/src/contracts/apiGateway/features/axiosRequest.ts

```diff
@@ -4,6 +4,8 @@ import { ApiGatewayContract } from '../apiGatewayContract';
 import { OutputType, RequestArguments } from '../types';
 import { getRequestParameters } from './requestParameters';
 
+// Naming cassé
+// Polymorphisme pour gérer axios et fetch dans la même "feature"
```

Comment:
Shouldn't this be in English?

Reply:
it should not have been merged, conversation for a next PR, i'll remove it